### PR TITLE
please add my stuffs

### PR DESCRIPTION
**Pull request by delraydavis**

### CODE REVIEW

File: go-ducks.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const consumables = [() => ''];
+const consumables = [() => 'SECRET'];
```
Comment (cwc-products) [inline go-ducks.js]: Please put this in app.js.

**Review by cwc-products — CHANGES_REQUESTED** [review]

File: go-ducks.js (hunk 1 above)
Comment (delraydavis) [reply go-ducks.js]: got it

Comment (delraydavis) [reply go-ducks.js]: text fails to render during app execution. Please provide further clarification

Comment (cwc-products) [reply go-ducks.js]: Please see #15

Comment (cwc-products) [reply go-ducks.js]: Obviously use your word and not "A"

Comment (delraydavis) [reply go-ducks.js]: Obviously its all working now ;)

**Comment by delraydavis** [comment]
Jeremy I need a review please.

**Review by cwc-products — CHANGES_REQUESTED** [review]
@delraydavis please link the original ticket to this PR.

**Review by papajeremy — APPROVED** [review]

**Comment by papajeremy** [comment]
This looks good.

**Comment by delraydavis** [comment]
conflicts are resolved.

**Review by cwc-products — CHANGES_REQUESTED** [review]
@delraydavis please run
````
npm run app
````

- [ ] confirm the word order is as expected.

**Comment by delraydavis** [comment]
word order corrected

**Comment by delraydavis** [comment]
Can I get a review on the merge resolution please?

File: go-ducks.js [hunk 2]
```diff
@@ -22,3 +22,4 @@ exports.goDucks = (function goDucks(messages = []) {
     }
   };
 })(consumables);
+
```
Comment (cwc-products) [inline go-ducks.js]: Please remove this extra space.

**Review by cwc-products — CHANGES_REQUESTED** [review]

**Comment by delraydavis** [comment]
extra space removed.

**Review by papajeremy — APPROVED** [review]
Space removed per request from christopher

**Review by lingyaintocareers — APPROVED** [review]
good

**Review by cwc-products — CHANGES_REQUESTED** [review]
@delraydavis please resolve most recent conflicts

**Comment by delraydavis** [comment]
resolved further conflict

**Review by cwc-products — CHANGES_REQUESTED** [review]
Run

````
npm run app
````

Please confim word order.

**Comment by delraydavis** [comment]
confirmed

**Review by cwc-products — APPROVED** [review]

**Pull request merged**